api/migrations/20210812174148-create-guild.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("guilds", {
+      id: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
```

Comment:
Should this be an UUID?

Reply:
The id of a guild is a Snowflake which is just a long string. It is not an id I generate myself.
https://discord.js.org/#/docs/main/stable/typedef/Snowflake